scrapy/squeues.py

```diff
@@ -122,23 +142,24 @@ def _pickle_serialize(obj):
         raise ValueError(str(e)) from e
 
 
+# queue.*Queue aren't subclasses of queue.BaseQueue
```

Comment:
I think `queuelib.queue.BaseQueue` needs to be a Protocol, not an ABC with `__subclasscheck__`.